src/components/elements/MenuLink.js

```diff
@@ -4,18 +4,21 @@ import { Link } from 'react-router-dom'
 import MenuItem from 'material-ui/MenuItem'
 import ActionDelete from 'material-ui/svg-icons/action/delete'
 
-export const MenuLink = (type, id, onClick, dispatch) => {
+export const MenuLink = (type, id, title, onClick, dispatch) => {
   var typeRoute = `/${type}s/${id}`
+  
   return (
     <MenuItem
       key={`${type}-${id}`}
       containerElement={<Link to={typeRoute} />}
-      primaryText={`${type} ${id}`}
+      primaryText={ticketPrimaryText(type, id, title)}
       rightIcon={<ActionDelete onClick={() => dispatch(onClick(id))} />}
     />
   )
 }
 
+const ticketPrimaryText = (type, id, title) => `${type} ${id}: ${title}`
```

Comment:
Probably `type` is not needed. It just takes up space, and it's probably going to be `ticket` always anyways.